deep-learning/_posts/2018-10-05-DenseNetsIsoIntenseInfantMRIBrainSegmentation.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: review
+title: "Exclusive Independent Probability Estimation using Deep 3D Fully
+Convolutional DenseNets for IsoIntense Infant Brain MRI Segmentation"
+tags: deep-learning segmentation brain DenseNets
```

Comment:
DenseNets is kind of too specific for a tag.  I would use the usual CNN.

Reply:
Done in 578fb93. Thanks for the review.